### PR TITLE
Updates README

### PR DESCRIPTION
**Pull request by rdcsg**
PT Story: https://www.pivotaltracker.com/story/show/128480189

Changes proposed in this pull request:
- added body text, title and stuff to the README.md file

Ready for review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+
+## Installing
+Fork and Clone this repo.
+
+Once the source is on your machine run the following commands to install all the necessary packages and gems:
+
+```
+$ bundle install
+
+$ rake db:create
```
Comment (diraulo) [inline README.md]: maybe go with `rails db:create` -- not really important since `rake` still works with Rails 5

**Comment by diraulo** [comment]
Looks good. This could be merged regardless of the comment about using the `rails` command

File: README.md [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+$ rake db:migrate
+
```
Comment (MikaelFeher) [inline README.md]: @rdcsg; 
- You have to close the backtricks that you opened on line 37.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+
+### Contributors
+[Mikael](https://github.com/MikaelFeher)
+[Lucia](https://github.com/luciademoja)
+[Lara](https://github.com/lollypop279
```
Comment (MikaelFeher) [inline README.md]: @rdcsg;
- You might want to close the parenthesis.
- Don't forget to remove the number 9.

**Comment by MikaelFeher** [comment]
@rdcsg;
Suggestion on the contributors section; to use our GH usernames instead of our actual names.

**Comment by rdcsg** [comment]
@MikaelFeher Hey I'm done now! :) Pls check new changes to see if ok. :)

**Comment by tochman** [comment]
I can only see one single commit. Where are your latest changes?

**Pull request merged**